### ISSUE TITLE
(feature request) add enable/disable

### ISSUE THREAD

**Issue by Ks89** [chrisbateman/webpack-visualizer#38]
I think that an option like 'enable' could be very useful.

plugins: [new Visualizer({
  filename: './statistics.html',
  **enable: true**
})],

**Comment by Dante-101** [comment]
You can very well insert it in the plugins array only when your `enable` expression evaluates to true?

**Comment by Ks89** [comment]
Yes, at the moment I used that solution, but a boolean (like in many other webpack plugins) could be useful.
It's only a suggestion :)
